packages/pn-personafisica-webapp/src/components/Contacts/__test__/CourtesyContacts.test.tsx

```diff
@@ -176,16 +213,16 @@ describe('CourtesyContacts Component', async () => {
       .reply(204);
     const result = render(
       <DigitalContactsCodeVerificationProvider>
-        <CourtesyContactItem type={CourtesyFieldType.PHONE} value={defaultPhoneAddress!.value} />
+        <CourtesyContacts contacts={digitalCourtesyAddresses} />
```

Comment:
replace digitalCourtesyAddresses with [defaultPhoneAddress]

Reply:
defaultPhoneAddress could be undefined, so I used [defaultPhoneAddress!]. Is that okay?

Reply:
yes